content/ADR-172-rentals.md

```diff
@@ -0,0 +1,88 @@
+
+The Land owner can accept an Offer by calling the `acceptOffer` function or safely transferring the Land to the Rentals contract with the offer data and signature. 
+
+A user that is interested in renting a Land can accept a Listing by calling the `acceptListing` function on the contract with the Listing data and signature.
+
+Successfully accepting any of these will start a Rental that will last as long as the Listing/Offer stipulated, MANA will be transferred from the tenant to the lessor, the rented Land will be transferred to the Rentals contract, and the tenant or any address determined by the tenant will be set as update operator of the Land to be able to deploy scenes.
+
+### Why is the Land transferred to the Rentals contract?
+
+This is to limit what the original owner of the asset can do with the Land, by transferring the ownership to the Rentals contract, the owner cannot interact with the asset directly anymore and have to use a set of functions defined by the Rentals smart contract to do so.
+
+For example, Once a rental starts, the owner can't do anything until the rental period ends. In that case, they can `claim` the Land back, rent it again the same way as before or set a different update operator to overwrite the one set by the tenant and prevent it's further use.
+
+### What happens when the rental ends?
+
+Nothing actively happens, if nothing is done by the owner, the update operator defined by the tenant can still deploy scenes. The owner can claim the Land back, rent it again or change the update operator to prevent the tenant from using it.
+
+### Can signatures be invalidated?
+
+A mechanism to do so was added to the Rentals contract as a safety measure.
+
+Listings and Offers are signed with a parameter called `indexes` made up of 3 different numbers used to check if it is still valid. 
+
+Contract Index: Can be increased by the owner of the rentals contract to invalidate all signatures.
+
+Signer Index: Can be increased to invalidate all signatures created by the user.
+
+Asset Index: Can be increased to invalidate all signatures created by the user for a certain Land.
+
+### Can Parcels inside a rented Estate be given individual update operator permissions?
+
+Yes.
+
+The Rentals contract exposes a function `setManyLandUpdateOperator` that can be called by the tenant (or the owner if the rental is over) to set the update operator of Parcels inside an Estate.
+
+This is also helpful because when the Estate is transferred for rental, internal Parcel permissions are not reset, so the tenant might need to call this function to prevent older tenants from overriding new deployments.
+
+## Solution Space Exploration
+
+The Rentals Smart Contract by itself might not be too easy to use, specially because handling signatures is a complex thing.
+
+In order for the contract to be used successfully, a dapp to facilitate signing and server to handle this signatures will be required.
```

Comment:
```suggestion
For the contract to be used successfully, it's required a dapp that facilitates signing and a server to handle these signatures.
```

Reply:
"it requires" seems a better fit than "it's required"

Reply:
SGTM